frontend/packages/dev-console/src/components/SearchPage.tsx

```diff
@@ -0,0 +1,40 @@
+import * as React from 'react';
+import { Button } from '@patternfly/react-core';
+import { useTranslation, Trans } from 'react-i18next';
+import { match as RMatch } from 'react-router';
+import { SearchPage } from '../../../../public/components/search';
+import NamespacedPage, { NamespacedPageVariants } from './NamespacedPage';
+import CreateProjectListPage from './projects/CreateProjectListPage';
+
+export interface SearchPageProps {
+  match: RMatch<{
+    ns?: string;
+  }>;
+  noProjectsAvailable?: boolean;
+}
+
+const Search: React.FC<SearchPageProps> = ({ noProjectsAvailable, ...props }) => {
+  const { t } = useTranslation();
+  const namespace = props.match.params.ns;
+  return (
+    <NamespacedPage variant={NamespacedPageVariants.light} hideApplications>
+      {namespace ? (
+        <SearchPage namespace={namespace} noProjectsAvailable={noProjectsAvailable} />
+      ) : (
+        <CreateProjectListPage title={t('devconsole~Search')}>
+          {(openProjectModal) => (
+            <Trans t={t} ns="devconsole">
+              Select a Project to search inside or{' '}
+              <Button isInline variant="link" onClick={openProjectModal}>
+                create a Project
+              </Button>
+              .
+            </Trans>
+          )}
+        </CreateProjectListPage>
+      )}
+    </NamespacedPage>
+  );
+};
+
+export default Search;
```

Comment:
Did we not want the `withStartGuide` here?

Reply:
Updated.
![1](https://user-images.githubusercontent.com/20013884/125096862-29545700-e0f3-11eb-95ae-62d64402f5bd.png)